features/step_definitions/helper_services.rb

```diff
@@ -714,3 +714,34 @@
   step 'the step should succeed'
   step 'the pod named "cluster-capacity" becomes ready'
 end
+# https://docs.openshift.com/container-platform/4.1/machine_management/applying-autoscaling.html
+# two steps, 1. create clusterautoscaler, follow by machineautoscaler
+# example of machine auto-scaler https://github.com/openshift-qe/output_references/blob/master/autoscale/machine-autoscaler.yaml
+Given /^I enable autoscaling for my cluster$/ do
```

Comment:
Sorry, I missed this earlier.

Is this a destructive step? It needs to check for the destructive tag.
Also I don't see clean-up logic.

Reply:
@akostadinov it's more of an enhancement step to ensure that it will automatically spin up more resources as needed.  I didn't put any clean up because it's meant to be permanent for the life-time of the cluster...run once and forget about it kind of deal.

Reply:
Then it needs to be inside `features/step_definitions/cluster_configuration/` directory. That directory we use for such kind of steps.

Reply:
@akostadinov ok, updated